scripts/install-latest.sh

```diff
@@ -107,16 +109,24 @@ yarn_install() {
 
   if [ -d "$HOME/.yarn" ]; then
     if [ -n `which yarn` ]; then
+      local SPECIFIED_VERSION
+      local VERSION_TYPE
       if [ "$1" = '--nightly' ]; then
         latest_url=https://nightly.yarnpkg.com/latest-tar-version
+        SPECIFIED_VERSION=`curl $latest_url`
```

Comment:
Please use lowercase for variables (I know, I messed this up in some files, I need to fix it 😛 ). Apparently uppercase is generally only used for environment variables.